src/Generator.h

```diff
@@ -492,11 +492,21 @@ class GeneratorRegistry {
 
 }  // namespace Internal
 
+namespace {
+// Return the address of a global with type T *. Never
+// assigned to. Used to assist the introspection framework.
```

Comment:
Do we explain (somewhere) in greater detail what this is used for? "used to assist" is a little thin.